docs/getting_started/intellij_plugin.md

```diff
@@ -0,0 +1,96 @@
+This plugin provides support for working with Chutney scenarios files in IntelliJ IDEA based IDEs.
```

Comment:
There is no link to this page.
Link it in mkdocs yaml configuration file.

Reply:
This page can be reach from /getting_started/run. I added a new subtitle "Chutney Idea plugin"